packages/edit-site/src/components/site-editor-routes/home-view.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import Editor from '../editor';
+import SidebarNavigationScreenMain from '../sidebar-navigation-screen-main';
+
+export const homeViewRoute = {
+	name: 'home-view',
+	match: ( params ) => {
+		return params.canvas !== 'edit';
+	},
+	areas: {
+		sidebar: <SidebarNavigationScreenMain />,
+		preview: <Editor />,
```

Comment:
I had forgotten why this didn't have a `mobile` area until I reviewed the `layout` component. Prepared https://github.com/WordPress/gutenberg/pull/66309 to document when each area is present. It also prevents the `edit` area from being rendered if the canvas mode is edit.